infrastructure/json/src/test/java/tech/pegasys/teku/infrastructure/json/JsonUtilTest.java

```diff
@@ -73,6 +73,18 @@ void getAttribute_getsAttributeAtParent() throws Exception {
     assertThat(result).contains(UInt64.valueOf(1234));
   }
 
+  @Test
+  void getAttribute_getsFirstAttribute() throws Exception {
```

Comment:
this testing should be expanded if we need this functionality
at least:
 - did not find requested field `"{\"data\": {}}"` and find slot for example.
 - field we're searching for is not first attribute - `{"data":{"a":"b", "c":"d"}}` - search c.
 - also above, search "d" and shouldn't be found?

Reply:
Removed since we explicitly specify the paths